### DISCUSSION TITLE
userstyles.org is terrible. Is there an alternative site?

### DISCUSSION THREAD

**Discussion by stamminator** [openstyles/stylus#1732] [General]
USO has only gotten worse over the years. Page loads frequently time out, the forums section is now gone, and it won't even load for me at all in Firefox anymore.

Similar to Stylus being a better and more open alternative to Stylish, is there a community repository in the same vein the we can use instead of USO? I'm grateful to the users who maintain READMEs with lists of decent userstyles, but these are hard to find and lack features like install counts, ratings, comments, etc.

**Comment by tophf** [comment]
Yes, use the links in the style manager and the built-in search in the popup:
* https://userstyles.world/explore
* https://uso.kkx.one/browse/styles
* https://greasyfork.org/en/scripts/?language=css

**Comment by stamminator** [comment]
After some more digging, I've found https://userstyles.world. I'm surprised at how long it took for me to find it. Probably just an oversight on my part, but I think it would be beneficial for this repo and wiki to push USW more prominently and visibly.

**Reply by tophf** [reply]
The alternative sites are already used by the style finder in the popup and emphasized in the style manager without styles, i.e. for the users who just installed the extension.
